tools/ratbagd.py

```diff
@@ -151,7 +152,30 @@ def _get_dbus_property(self, property):
 
     def _set_dbus_property(self, property, type, value):
         # Sets a cached property on the bus.
+
+        # Take our real value and wrap it into a variant. To call
+        # org.freedesktop.DBus.Properties.Set we need to wrap that again
+        # into a (ssv), where v is our value's variant.
+        # args to .Set are "interface name", "function name",  value-variant
```

Comment:
it took me long enough to figure out how to send this, so I'd rather have the comment there to clarify what the code does ;)